flowapi/flowapi/config.py

```diff
@@ -43,11 +43,13 @@ def get_config():
     )
     flowdb_host = os.environ["FLOWDB_HOST"]
     flowdb_port = os.environ["FLOWDB_PORT"]
+    server_id = getsecret("FLOWAPI_IDENTIFIER", os.environ["FLOWAPI_IDENTIFIER"])
```

Comment:
Nitpick, but can we call the variable `flowapi_identifier` or `flowapi_server_id`, so that it's clear which server this is referring to? Mainly to make the line below (which sets `JWT_DECODE_AUDIENCE`) more understandable at a glance.